lib/dish.rb

```diff
@@ -0,0 +1,14 @@
+class Dish
+  attr_reader :name, :price, :hash
+
+  def initialize(name = "Hash brown", price = 0.99)
+    @name = name
+    @price = price
+    @hash = {}
```

Comment:
I'm not sure I understand why `@hash` is there — it you need to transform the object into a key/value pair (what `itemise` is doing), you could maybe do it in the class that will call `Dish` and transform the list of dishes into a hash. But it somehow feels weird to have this hash into the Dish class, since a single Dish instance will translate only into one key/value pair (let me know if that's not clear!)

Reply:
Or if you want to keep the same behaviour, you could simply by rewriting the method `itemise` like this:
```ruby
def itemise
   { @name => @price.round(2) }
end
```

so you don't need to `@hash` attribute at all